companies/porkbun.json

```
@@ -0,0 +1,26 @@
+{
+    "slug": "porkbun",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "telecommunication"
+    ],
+    "name": "Porkbun",
+    "address": "21370 SW Langer Farms Parkway, Suite 142-429\nSherwood, OR 97140, US",
```

```suggestion
    "address": "21370 SW Langer Farms Parkway, Suite 142-429\nSherwood, OR 97140\nUnited States",
```
fixed address format

---

Almost, we've decided on "United States of America". :)

```suggestion
    "address": "21370 SW Langer Farms Parkway\nSuite 142-429\nSherwood, OR 97140\nUnited States of America",
```